spec/lib/canvas/checks/valid_json_check_spec.rb

```diff
@@ -15,11 +15,11 @@
     it "adds an offense when a file contains invalid json" do
       copy_example_directory("vagabond")
       subject.run
-      message = "unexpected token at 'This is an invalid custom type."
+      message_pattern = /Invalid JSON: .+ - \nunexpected \w+(?:: | at )'This is an invalid custom type\.\n'/
```

Comment:
I guess this is highlighting that we don't actually control this error message, its coming from the JSON parser. Should we just assert that there is an offense, and we parrot back the invalid content.

Reply:
Yeah exactly, I wasn't really sure what level of confidence we need here so just wanted to make the spec work, think we can look later if we need to lessen further but given its working now and we barely touch this I'll just leave